test/unit/cli_test.rb

```diff
@@ -117,7 +117,8 @@ class CliTest < Minitest::Test
         root_path: @temp_dir,
         load_paths: ["path"],
         package_paths: "**/",
-        custom_associations: ["cached_belongs_to"]
+        custom_associations: ["cached_belongs_to"],
+        inflections_file: "config/inflections.yml"
```

Comment:
Along with [the lines below](https://github.com/Shopify/packwerk/pull/53/files#diff-efc360b012815c4277ebe4bf8fdde379c5596bacc8dbc6cc139419abeac71c54R139), this key is now required in the `RunContext` because we now gotta know the inflection file path.